project/project.json5

```diff
@@ -0,0 +1,238 @@
+            {
+                type: 'subsector',
+                class: 'Subsector',
+                name: 'DECARB 2023 Demand Subsectors',
+                display_name: 'Subsector',
+                description: 'DECARB 2023 Demand Subsectors; No breakouts for buildings, vehicles by class and range, 3-digit NAICS for most manufacturing',
+                file: 'dimensions/subsectors.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'metric',
+                class: 'EnergyEndUse',
+                name: 'DECARB 2023 End Uses',
+                display_name: 'End Use',
+                description: 'DECARB 2023 Demand End Uses', // TODO: Describe contents--level of detail, fuel types
+                file: 'dimensions/enduses.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'model_year',
+                class: 'ModelYear',
+                name: 'DECARB 2023 Model Years',
+                display_name: 'Model Year',
+                description: 'DECARB 2023 Model Years; Biennial 2020-2030, then every 5 years to 2050',
+                file: 'dimensions/model_years.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'scenario',
+                class: 'Scenario',
+                name: 'DECARB 2023 High-Resolution Demand Scenarios',
+                display_name: 'Scenario',
+                description: 'DECARB 2023 High-Resolution Demand Scenarios; Limited to a single, high decarbonization scenario',
+                file: 'dimensions/scenarios.csv',
+                module: 'dsgrid.dimension.standard',
+            },
+            {
+                type: 'time',
+                class: 'Time',
+                frequency: 'P0DT1H0M0.000000S',
+                name: 'Time-2018-EST-hourly-periodBeginning-noLeapDayAdjustment-total',
+                display_name: 'Time EST',
+                time_type: 'datetime',
+                leap_day_adjustment: 'none',
+                description: 'Time dimension, 2018 hourly EST, hour-beginning, no Leap Day Adjustment, total value',
+                time_interval_type: 'period_beginning',
+                str_format: '%Y-%m-%d %H:%M:%S',
+                timezone: 'EasternStandard',
+                measurement_type: 'total',
+                ranges: [
+                    {
+                        start: '2018-01-01 00:00:00',
+                        end: '2018-12-31 23:00:00',
+                    },
+                ],
+            },
+            {
+                type: 'weather_year',
+                class: 'WeatherYear',
+                name: '2018 Weather Year',
+                display_name: 'Weather 2018',
+                description: 'Weather year, 2018 only',
+                file: 'dimensions/weather_years.csv',
+                module: 'dsgrid.dimension.standard',
+            },
```

Comment:
Expecting timestamps and weather to follow calendar year 2018, as seen from the EST timezone. dsgrid can perform end/beginning-of-year time-wrapping and shifting from period-ending to period-beginning timestamps as needed to ingest datasets. We also support TEMPO's representative time. Please let us know if you have any time conversion questions/other time formats we need to support.

Reply:
For buildings we will be using the time conventions from ResStock/ComStock EUSS, so hopefully that doesn't require additional formats/conversions.